symphony-bdk-core-invokers/symphony-bdk-core-invoker-jersey2/src/main/java/com/symphony/bdk/core/api/invoker/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -0,0 +1,171 @@
+package com.symphony.bdk.core.api.invoker.jersey2;
+
+import com.symphony.bdk.core.api.invoker.ApiClient;
+import com.symphony.bdk.core.api.invoker.ApiClientBuilder;
+import org.glassfish.jersey.SslConfigurator;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.client.ClientProperties;
+import org.glassfish.jersey.client.HttpUrlConnectorProvider;
+import org.glassfish.jersey.jackson.JacksonFeature;
+import org.glassfish.jersey.media.multipart.MultiPartFeature;
+
+import javax.net.ssl.SSLContext;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+
+/**
+ * Specific implementation of {@link ApiClientBuilder} which creates a new instance of an {@link ApiClientJersey2}.
+ */
+public class ApiClientBuilderJersey2 implements ApiClientBuilder {
+
+  private String basePath;
+  private String keyStorePath;
+  private String keyStorePassword;
+  private String trustStorePath;
+  private String trustStorePassword;
+  private Map<String, String> defaultHeaders;
+  private int connectionTimeout;
+  private int readTimeout;
+  private String temporaryFolderPath;
+
+  public ApiClientBuilderJersey2() {
+    basePath = "https://acme.symphony.com";
+    keyStorePath = null;
+    keyStorePassword = null;
+    trustStorePath = null;
+    trustStorePassword = null;
+    defaultHeaders = new HashMap<>();
+    connectionTimeout = 0;
+    readTimeout = 0;
+    temporaryFolderPath = null;
+    userAgent("Symphony BDK/2.0/java");
```

Comment:
Maybe we could try to be close the "standard" for user-agent (https://stackoverflow.com/questions/2601372/what-is-the-standard-format-for-a-browsers-user-agent-string)
So maybe something like _Symphony BDK/2.0 java/1.8.212_ ?

Reply:
In the future that would be nice to retrieve the real version from the `pom.xml`, potentially by using Maven resources filtering (cf. https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html). Could you create a story for that?